standards/qa-process.md

```diff
@@ -19,13 +19,13 @@
 - PM or Lead Dev will create the “QA Sprint” issue with:
   - Short project or feature description
   - Delivery/testing timeframe
-  - Associated milestone
+  - Associated milestone containing Definition of Done
   - Associated project board
   - Sandbox URL/ Testflight App Name
   - App Version number(s) to be tested
   - Devices / Browsers to be tested
   - Definition of Done
-  - Acceptance Test in Markdown Format.
+  - Acceptance test in Markdown Format. An acceptance test will be necessary for all new projects. If you would like a tutorial on creating an acceptance test please contact qa@bitwiseindustries.com and we can set up a meeting to get you started.
```

Comment:
I would suggest creating an acceptance test defined on another page to reduce the number of requests QA gets in the long run. 

For now this is not a blocker, but should probably be on our radar. @dcedano do you know if we already have docs in place for acceptance testing?